docs/docsite/rst/galaxy/user_guide.rst

```diff
@@ -364,9 +364,27 @@ Dependencies
 ------------
 
 Roles can also be dependent on other roles, and when you install a role that has dependencies, those dependencies will automatically be installed.
+When dependencies are encountered by ``ansible-galaxy``, it will automatically install each dependency to the ``roles_path``.
 
-You specify role dependencies in the ``meta/main.yml`` file by providing a list of roles. If the source of a role is Galaxy, you can simply specify the role in
-the format ``namespace.role_name``. You can also use the more complex format in ``requirements.yml``, allowing you to provide ``src``, ``scm``, ``version``, and ``name``.
+There are two ways to define the dependencies of a role:
+
+Using ``meta/requirements.yml``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`.. versionadded:: 2.10`
+
+You can create the file ``meta/requirements.yml`` and define dependencies in the same format used for :file:`requirements.yml` described in the `Installing multiple roles from a file`_ section.
+
+From there, you can import or include the specified roles in your tasks.
+
+Using ``meta/main.yml``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Alternatively, you can also specify role dependencies in the ``meta/main.yml`` file by providing a list of roles under the ``dependencies`` section. If the source of a role is Galaxy, you can simply specify the role in
```

Comment:
```suggestion
Alternately, you can  specify role dependencies in the ``meta/main.yml`` file by providing a list of roles under the ``dependencies`` section. If the source of a role is Galaxy, you can simply specify the role in
```

Reply:
Not a native english speaker but it seems that alternatively was a bit more what I was trying to convey:

> Alternately means several things taken in turn, consecutively, one after another. Alternately is an adverb formed by adding -ly to the word alternate, which comes from the Latin word alternatus meaning one after the other. Alternatively means another choice, on the other hand, another possibility.